Classes/Controller/CompanyController.php

```diff
@@ -78,10 +78,8 @@ public function initializeAction(): void
         }
     }
 
-    /**
-     * @Validate("String", param="letter")
-     * @Validate("StringLength", param="letter", options={"minimum" = 0, "maximum" = 3})
-     */
+    #[Validate(['validator' => 'String', 'param' => 'letter'])]
```

Comment:
Sorry. I can't comment in unchanged code. Can you please order the use-imports

Reply:
Done ☑️